chart/templates/operator.yaml

```diff
@@ -264,20 +264,19 @@ subjects:
   name: addon-status-sa
   namespace: open-cluster-management
 ---
-apiVersion: apps/v1
 kind: Deployment
+apiVersion: apps/v1
 metadata:
+  name: status-controller
   labels:
-    app: status-controller
+    control-plane: controller-manager
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/instance: controller-manager
     app.kubernetes.io/component: manager
     app.kubernetes.io/created-by: status-addon
-    app.kubernetes.io/instance: controller-manager
-    app.kubernetes.io/managed-by: kustomize
-    app.kubernetes.io/name: deployment
     app.kubernetes.io/part-of: status-addon
-    control-plane: controller-manager
-  name: addon-status-controller
```

Comment:
Why all the re-ordering here? It was alphabetical, isn't that good enough?

Reply:
It was not intentional, I think it was regenerated by make chart.
As I don't care, I reverted back to the original order.